### ISSUE TITLE
unknown postgres type jsonb

### ISSUE THREAD

**Issue by swuecho** [looker-open-source/malloy#282]
jsonb is a pretty common data type nowadays.

**Comment by lloydtabb** [comment]
Yes.  We want to somehow expose the schema of these automatically.  Not sure if that is possible.  At a minimum you should be able to declare the structure and use them like nested object...
